src/Uno.UI/UI/Xaml/Controls/SwipeControl/SwipeItem.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// IconSource is implemented in WUX in the OS repo, so we don't need to
+// include IconSource.h on that side.
+
+// Imported in uno on 2021/03/21 from commit 307bd99682cccaa128483036b764c0b7c862d666
+// https://github.com/microsoft/microsoft-ui-xaml/blob/307bd99682cccaa128483036b764c0b7c862d666/dev/SwipeControl/SwipeItem.cpp
+
+using System;
+using System.Windows.Input;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class SwipeItem : FrameworkElement
```

Comment:
Is it necessary to inherit from `FrameworkElement`? Just noticed that it should really just be a `DependencyObject` (actually when I was wondering about the `new Background` stuff).

If it's necessary, add an 'Uno workaround' comment and an explanation.

Reply:
Good point. Not even sure why it was a `FrameworkElement` ... most probably for bindings, but it was not a good reason so I've converted it to a DO.